container/container_unix.go

```diff
@@ -485,16 +486,18 @@ func (container *Container) CopyImagePathContent(v volume.Volume, destination st
 		return err
 	}
 
-	path, err := v.Mount()
+	id := stringid.GenerateNonCryptoID()
```

Comment:
We generally use non-crypto for these sort of IDs (even container ID's are non-crypto). These are going to be relatively short-lived and not likely to conflict (and not catastrophic if they happen to).